Formula/f/ffmpeg.rb

```diff
@@ -6,6 +6,7 @@ class Ffmpeg < Formula
   # None of these parts are used by default, you have to explicitly pass `--enable-gpl`
   # to configure to activate them. In this case, FFmpeg's license changes to GPL v2+.
   license "GPL-2.0-or-later"
+  revision 1
```

Comment:
`ffmpeg` no longer depends on `openvino` after https://github.com/Homebrew/homebrew-core/pull/171740

Reply:
Ok, thank you